src/host/main.cpp

```diff
@@ -404,18 +430,23 @@ int main(int argc, char** argv)
     startup_config.network = config.network;
     startup_config.worker_threads = config.worker_threads;
     startup_config.node_certificate = config.node_certificate;
-    startup_config.attestation = config.attestation;
+    startup_config.attestation.snp_endorsements_servers =
+      config.attestation.snp_endorsements_servers;
 
-    // Get the nodes security policy via environment variable
-    if (access(ccf::pal::snp::DEVICE, F_OK) == 0)
+    if (config.attestation.environment.security_policy.has_value())
     {
-      LOG_INFO_FMT("Warning: AMD SEV-SNP support is currently experimental");
-      auto policy = std::getenv("SECURITY_POLICY");
-      if (policy != nullptr)
-      {
-        std::vector<uint8_t> raw = crypto::raw_from_b64(policy);
-        startup_config.security_policy = std::string(raw.begin(), raw.end());
-      }
+      startup_config.attestation.environment.security_policy =
+        read_required_environment_variable(
+          config.attestation.environment.security_policy.value(),
+          "attestation security policy");
+    }
+
+    if (config.attestation.environment.uvm_endorsements.has_value())
+    {
+      startup_config.attestation.environment.uvm_endorsements =
+        read_required_environment_variable(
+          config.attestation.environment.uvm_endorsements.value(),
+          "UVM endorsements");
     }
```

Comment:
Note that this is currently unused. The work to parse, verify and store this will be done in https://github.com/microsoft/CCF/issues/4193.